examples/platform/telink/common/src/AppTaskCommon.cpp

```diff
@@ -157,7 +157,8 @@ class AppFabricTableDelegate : public FabricTable::Delegate
     {
         if (chip::Server::GetInstance().GetFabricTable().FabricCount() == 0)
         {
-            bool isCommissioningFailed = chip::Server::GetInstance().GetCommissioningWindowManager().IsCommissioningWindowOpen();
+            bool isBasicCommissioningMode = chip::Server::GetInstance().GetCommissioningWindowManager().GetCommissioningMode() ==
+                Dnssd::CommissioningMode::kEnabledBasic;
```

Comment:
Why does basic vs enhanced matter here?

If commissioner A opens a basic commissioning window, commissioner B is trying to commission, and commissioner A removes itself during that time... why would we want to reset mid-commissioning?